build_tools/mac_notarize.sh

```diff
@@ -13,17 +13,16 @@ for INPUT in "$@"; do
     echo "Processing $INPUT"
     test -f "$INPUT" || die "Not a file: $INPUT"
     ext="${INPUT##*.}"
-    (test "$ext" = "zip" || test "$ext" = "pkg") || die "Unrecognized extension: $ext"
+    { test "$ext" = "zip" || test "$ext" = "pkg"; } || die "Unrecognized extension: $ext"
 
     xcrun notarytool submit "$INPUT" --keychain-profile AC_PASSWORD --wait
 
     if test "$ext" = "zip"; then
         TMPDIR=$(mktemp -d)
         echo "Extracting to $TMPDIR"
         unzip -q "$INPUT" -d "$TMPDIR"
-        # Force glob expansion.
-        STAPLE_TARGET="$TMPDIR"/*
-        STAPLE_TARGET=$(echo $STAPLE_TARGET)
+        STAPLE_TARGET_ARRAY=("$TMPDIR"/*)
+        STAPLE_TARGET="${STAPLE_TARGET_ARRAY[*]}"
```

Comment:
this script is valid POSIX shell; I wouldn't want to use Bash extensions without a good reason.
I'd only fix the inadvertent word splitting by writing `STAPLE_TARGET=$(echo "$TMPDIR"/*)`

But it's fine as-is, this is not a blocker